### PR TITLE
implement getStatistic

### PR DESCRIPTION
**Pull request by aksoli666**

### CODE REVIEW

**Review by Rommelua — CHANGES_REQUESTED** [review]
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected

image

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+                if (operation.equals("supply")) {
+                    sumSupply += digit;
+                } else if (operation.equals("buy")) {
+                    sumBuy += digit;
+                }
+            }
+        } catch (IOException exception) {
+            throw new RuntimeException("I can't perform the operation!", exception);
+        }
+        String report = createReport(sumSupply, sumBuy);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/WorkWithFile.java:29]:
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
```
`readFile` can return `String `or `List<String>` (your choice)

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
+        report.append("buy,").append(sumBuy).append(System.lineSeparator());
+        report.append("result,").append(result).append(System.lineSeparator());
+        return report.toString();
+    }
 
+    public void writeToFile(String toFileName, String report) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException exception) {
+            throw new RuntimeException("I can`t do this operation!", exception);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/WorkWithFile.java:46]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/WorkWithFile.java (hunk 1 above)
Comment (aksoli666) [reply src/main/java/core/basesyntax/WorkWithFile.java:29]: i don`t quite understand what needs to be done

**Pull request closed without merging**